tests/impl.cpp

```diff
@@ -1540,6 +1543,25 @@ bool test_mm_mul_ps(const float *_a, const float *_b)
     return validateFloat(c, dx, dy, dz, dw);
 }
 
+bool test_mm_dp_ps(const float *_a, const float *_b)
+{
+    const int imm = 0xFF;
```

Comment:
Why changing the value from `0x3F` to `0xFF`?

Reply:
I wanted to validate the special case `0xFF`. Since the last parameter of `_mm_dp_ps` is `imm8`, only constants can be passed, that prevents me from writing test cases with better coverage though.

Reply:
Each occurrence of `int imm` in `sse2neon.h` has the same restriction to comply with Intel SSE intrinsics.